### PR TITLE
Fix parsing peak usage when value is not set

### PR DESCRIPTION
**Pull request by dupondje**
Sometimes peak usage is not yet visible in the 13 months history code. It gives a 0.0kW value with timestamp (632525252525W).

This commit fixes handling this case.

### CODE REVIEW

File: dsmr_parser/objects.py [hunk 1]
```diff
@@ -183,9 +183,19 @@ def unit(self):
         return self.values[2]['unit']
 
     def __str__(self):
+        timestamp = self.datetime
+        if isinstance(self.datetime, datetime.datetime):
```
Comment (jbouwh) [inline dsmr_parser/objects.py:187]:
```suggestion
        if isinstance(datetime, datetime.datetime):
```

File: dsmr_parser/objects.py [hunk 2]
```diff
@@ -183,9 +183,19 @@ def unit(self):
...
+            timestamp = self.datetime.astimezone().astimezone(pytz.utc).isoformat()
+        timestamp_occurred = self.occurred
+        if isinstance(self.occurred, datetime.datetime):
```
Comment (jbouwh) [inline dsmr_parser/objects.py:190]:
```suggestion
        if isinstance(timestamp_occurred, datetime.datetime):
```

File: dsmr_parser/objects.py [hunk 3]
```diff
@@ -183,9 +183,19 @@ def unit(self):
...
+            timestamp_occurred = self.occurred.astimezone().astimezone(pytz.utc).isoformat()
+        value = self.value
+        if isinstance(self.value, datetime.datetime):
```
Comment (jbouwh) [inline dsmr_parser/objects.py:193]:
```suggestion
        if isinstance(value, datetime.datetime):
```

File: dsmr_parser/objects.py [hunk 4]
```diff
@@ -183,9 +183,19 @@ def unit(self):
...
+            value = self.value.astimezone().astimezone(pytz.utc).isoformat()
```
Comment (jbouwh) [inline dsmr_parser/objects.py:194]:
```suggestion
            value = value.astimezone().astimezone(pytz.utc).isoformat()
```

File: dsmr_parser/objects.py [hunk 5]
```diff
@@ -183,9 +183,19 @@ def unit(self):
...
+        if isinstance(self.value, Decimal):
+            value = float(self.value)
         output = "{}\t[{}] at {} occurred {}"\
-            .format(str(self.value), str(self.unit), str(self.datetime.astimezone().astimezone(pytz.utc).isoformat()),
-                    str(self.occurred.astimezone().astimezone(pytz.utc).isoformat()))
+            .format(str(value), str(self.unit), str(timestamp), str(timestamp_occurred))
```
Comment (jbouwh) [inline dsmr_parser/objects.py:198]: I suggest using an f-formatted string here

**Review by jbouwh — CHANGES_REQUESTED** [review]
Some minor suggestions added

**Review by ejpalacios — CHANGES_REQUESTED** [review]
The `__str__` method of `MBusObject` can also cause problems when the current power peak timestamp of `BELGIUM_MAXIMUM_DEMAND_MONTH` is not initialised. 
E.g., `1-0:1.6.0(632525252525W)(00.000*kW)`.

I will suggest handling this in the same way as the monthly peaks.

File: dsmr_parser/objects.py (hunk 5 above)
Comment (dupondje) [reply dsmr_parser/objects.py:201]: f-format is nowhere used in the code atm. Let's keep this for now and maybe do a follow-up PR to replace then all at once?

**Pull request merged**